src/main/java/com/idrsolutions/microservice/utils/LibreOfficeHelper.java

```diff
@@ -76,4 +92,48 @@ public static boolean convertToPDF(final String sofficePath, final File file, fi
         return true;
     }
 
+    /**
+     * Converts an office file to PDF using the specified LibreOffice executable.
+     *
+     * @param sofficePath The path to the soffice executable
+     * @param file The office file to convert to PDF
+     * @param uuid The uuid of the conversion on which to set the error if one occurs
+     * @return 0 if successful, 1050 if libreoffice timed out, or 1070 if libreoffice has an internal error
+     * occurs
+     */
+    public static Result convertDocToPDF(final String sofficePath, final File file, final String uuid) {
```

Comment:
Could we have the convertToPDF method call convertDocToPDF to reduce code duplication?